types/etherport-client/index.d.ts

```diff
@@ -0,0 +1,10 @@
+export interface EtherPortClientConstructorArgs {
+    host: string;
+    port: number;
+    reconnectTimeout?: number;
+}
+
+// tslint:disable-next-line:no-unnecessary-class
+export class EtherPortClient {
```

Comment:
I believe this class should extend `EventEmitter`: <https://github.com/mwittig/etherport-client/blob/939cd5f24c397bc424cb1ab39d3c9df578e19dd6/index.js#L33>